packages/sdk-components-react-radix/src/tabs.ws.ts

```diff
@@ -0,0 +1,197 @@
+  tc.mt(2),
+  tc.focusVisible(
+    [tc.outline("none"), tc.ring("ring", 2, "background", 2)].flat()
+  ),
+].flat();
+
+export const metaTabs: WsComponentMeta = {
+  category: "radix",
+  type: "container",
+  label: "Tabs",
+  icon: RadioCheckedIcon,
+  template: [
+    {
+      type: "instance",
+      component: "Tabs",
+      dataSources: {
+        tabsValue: { type: "variable", initialValue: "0" },
+      },
+      props: [
+        { type: "dataSource", name: "value", dataSourceName: "tabsValue" },
+        {
+          name: "onValueChange",
+          type: "action",
+          value: [
+            {
+              type: "execute",
+              args: ["value"],
+              code: `tabsValue = value`,
+            },
+          ],
+        },
+      ],
+      children: [
+        {
+          type: "instance",
+          component: "TabsList",
+          // inline-flex h-10 items-center justify-center rounded-md bg-muted p-1 text-muted-foreground
+          styles: [
+            tc.inlineFlex(),
+            tc.h(10),
+            tc.items("center"),
+            tc.justify("center"),
+            tc.rounded("md"),
+            tc.bg("muted"),
+            tc.p(1),
+            tc.text("mutedForeground"),
+          ].flat(),
+          children: [
+            {
+              type: "instance",
+              component: "TabsTrigger",
+              children: [
+                {
+                  type: "instance",
+                  component: "Button",
+                  styles: tabsTriggerStyles,
+                  children: [{ type: "text", value: "Account" }],
+                },
+              ],
+            },
+            {
+              type: "instance",
+              component: "TabsTrigger",
+              children: [
+                {
+                  type: "instance",
+                  component: "Button",
+                  styles: tabsTriggerStyles,
+                  children: [{ type: "text", value: "Password" }],
+                },
+              ],
+            },
+          ],
+        },
+        {
+          type: "instance",
+          component: "TabsContent",
+          styles: tabsContentStyles,
+          children: [
+            { type: "text", value: "Make changes to your account here." },
+          ],
+        },
+        {
+          type: "instance",
+          component: "TabsContent",
+          styles: tabsContentStyles,
+          children: [{ type: "text", value: "Change your password here." }],
+        },
+      ],
+    },
+  ],
+};
+
+export const metaTabsList: WsComponentMeta = {
+  category: "hidden",
+  detachable: false,
+  type: "container",
+  label: "Tabs List",
+  icon: RadioCheckedIcon,
+  requiredAncestors: ["Tabs"],
+};
+
+export const metaTabsTrigger: WsComponentMeta = {
+  category: "hidden",
+  type: "container",
+  label: "Tabs Trigger",
+  icon: RadioCheckedIcon,
+  requiredAncestors: ["TabsList"],
+  invalidAncestors: ["TabsTrigger"],
```

Comment:
TabTrigger

Reply:
Tabs is a feature namespace. Called the same in radix. Consistent with all other components.